test/classes/delete-free/lifetimes/bad-global-borrows-block.good

```diff
@@ -1,4 +1,3 @@
 bad-global-borrows-block.chpl:7: error: Scoped variable globalBorrow would outlive the value it is set to
 bad-global-borrows-block.chpl:6: note: consider scope of myOwned
 bad-global-borrows-block.chpl:12: error: Scoped variable globalBorrow would outlive the value it is set to
-bad-global-borrows-block.chpl:11: note: consider result here
```

Comment:
I know you mentioned it in the PR message, but this seems... odd.  And unfortunate?